skimage/feature/corner.py

```diff
@@ -122,9 +122,13 @@ def hessian_matrix(image, sigma=1, mode='constant', cval=0):
         weighting function for the auto-correlation matrix.
     mode : {'constant', 'reflect', 'wrap', 'nearest', 'mirror'}, optional
         How to handle values outside the image borders.
-    cval : float, optional
+    cval : float, 'F'
+        else:optional
         Used in conjunction with mode 'constant', the value outside
         the image boundaries.
+    order : {'C', 'F'}, optional
+        this parameter allows for the use of reverse or forward order of
```

Comment:
this -> This. Also, no need to specify "this parameter". How about:

It just occurred to me, that, even though in the discussion in #2301 we settled on C|F, actually it's not semantically correct: the order is reversed, not simply transposed. So I suggest changing this to `order : {'xy', 'rc'}`. @ahojnnes thoughts?